src/test/tests/graphics/scenery/tests/examples/advanced/CurveGeometryExample.kt

```diff
@@ -0,0 +1,102 @@
+package graphics.scenery.tests.examples.PDB
```

Comment:
This should live in the `graphics.scenery.tests.examples.basic` (or advanced, your call) package.